OfferteApp/OfferteApp.csproj

```diff
@@ -5,17 +5,20 @@
         <Nullable>enable</Nullable>
         <ImplicitUsings>enable</ImplicitUsings>
         <InvariantGlobalization>true</InvariantGlobalization>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
     </PropertyGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.AspNetCore.OpenApi" Version="8.0.1"/>
+        <PackageReference Include="BCrypt.Net-Next" Version="4.0.3" />
```

Comment:
waarom deze library

Reply:
Om te encrypten?

Reply:
In mijn vorige project waren het oprecht drie regels dus daar ga je geen library voor installeren.